xbmc/music/tags/MusicInfoTag.h

```diff
@@ -258,6 +261,7 @@ class CMusicInfoTag final : public IArchivable, public ISerializable, public ISo
   std::string m_stationName;
   std::string m_stationArt; // Used to fetch thumb URL for Shoutcasts
   std::string m_songVideoURL; // link to a video for a song
+  std::map<int, chapterDetails> m_chapters; // Ch No., name, start time, end time
```

Comment:
Seeing `std::map<int, chapterDetails>`three times in this header alone, it could make sense to introduce a type for it, to make code more readable: `using ChapterDetailsMap = std::map<int, chapterDetails>;`, in Song.h, near definition of `chapterDetails` struct.